cfme/infrastructure/provider/__init__.py

```diff
@@ -171,7 +171,7 @@ def __init__(
         self.key = key
         self.provider_data = provider_data
         self.zone = zone
-        self.vm_name = version.pick({version.LOWEST: "VMs", '5.5': "VMs and Instances"})
+        self.vm_name = deferred_verpick({version.LOWEST: "VMs", '5.5': "VMs and Instances"})
```

Comment:
Btw this was necessary in case you want to list providers in yaml using `utils.providers.new_list_providers` / `utils.providers.list_providers` without having an appliance connection which IMHO shouldnt be required, if you're just playing around in ipython and trying to get provider creds or whatever, as people do.